src/md/src/InstallationSteps/DeployReportPortal.md

```diff
@@ -26,11 +26,18 @@ Where:
 - **docker logs &lt;container_name&gt;** shows logs from selected container
 - **docker ps -a | grep "reportportal_" | awk '{print $1}' | xargs docker rm -f** Deletes all ReportPortal containers
 
-4. Open your browser with the IP address of the deployed environment at port **8080**
+4. Open your browser with the IP address of the deployed environment at port **9090**
+
+You can get the host IP address by using the following docker command:
+
+```shell
+ $ docker-machine ip default
+```
 
   ```
-  http://IP_ADDRESS:8080
+  http://IP_ADDRESS:9090
```

Comment:
Guys, please, do not change default IP. 
Users got used to previous one

Reply:
@AlehB

Reply:
@avarabyeu , it was done according to the latest docker-compose.yml provided by @pbortnik
if we are going to keep 8080 and 8081 ports, let's make the same in the documentation